### PR TITLE
Add offset as command line argument

### PR DESCRIPTION
**Pull request by Vesihiisi**
Add `--offset <int>` as a command line argument in order to read the database table with a specific offset.

Task: https://phabricator.wikimedia.org/T162659

### CODE REVIEW

File: importer/importer.py [hunk 1]
```diff
@@ -43,8 +43,17 @@ def __init__(self, countryname, languagename):
...
+def make_query(specific_table, offset):
+    """
+    Generate a query to retrieve data from database.
+
+    :param specific_table: Name of table to retrieve data from.
+    :param offset: Optional offset to start from.
+    """
+    query = 'select DISTINCT *  from `{}`'.format(specific_table)
+    if isinstance(offset, int):
+        query = query + " LIMIT {}, 18446744073709551615;".format(str(offset))
```
Comment (lokal-profil) [inline importer/importer.py]: `query = query + " LIMIT..."` can be replaced by `query += " LIMIT..."`

18446744073709551615:
If this is just an arbitrary large number then I would make it a variable called something like that to clarify.

File: importer/importer.py [hunk 2]
```diff
@@ -43,8 +43,17 @@ def __init__(self, countryname, languagename):
...
-def make_query(specific_table):
-    return ('select DISTINCT *  from `{}`').format(specific_table)
+def make_query(specific_table, offset):
+    """
+    Generate a query to retrieve data from database.
+
+    :param specific_table: Name of table to retrieve data from.
+    :param offset: Optional offset to start from.
+    """
+    query = 'select DISTINCT *  from `{}`'.format(specific_table)
```
Comment (lokal-profil) [inline importer/importer.py]: Is the double space on purpose?

Comment (Vesihiisi) [inline importer/importer.py]: Nope, didn't see it! Out it goes.

File: importer/importer.py (hunk 1 above)
Comment (Vesihiisi) [inline importer/importer.py]: Apparently, in MySQL the only way to use OFFSET is as an argument of LIMIT, and then you have to specify a limit as well. [It's even in the documentation](https://dev.mysql.com/doc/refman/5.7/en/select.html) so, seriously, yeah (ctrl + f 18446744073709551615)

File: importer/importer.py [hunk 3]
```diff
@@ -50,9 +50,10 @@ def make_query(specific_table, offset):
     :param specific_table: Name of table to retrieve data from.
     :param offset: Optional offset to start from.
     """
-    query = 'select DISTINCT *  from `{}`'.format(specific_table)
+    query = 'select DISTINCT * from `{}`'.format(specific_table)
     if isinstance(offset, int):
-        query = query + " LIMIT {}, 18446744073709551615;".format(str(offset))
+        unlimited = "18446744073709551615" # workaround for MySQL requiring a limit when using offset
```
Comment (lokal-profil) [inline importer/importer.py]: I think this line might bow be too long (i.e. >79 chars). Also two spaces before comment-`#`.

Comment (lokal-profil) [inline importer/importer.py]: HUm.. I would have expected that the CI should pick up the two spaces bit

Comment (Vesihiisi) [inline importer/importer.py]: Done.

**Pull request merged**